config/helpers.php

```diff
@@ -615,14 +528,12 @@ function svg($file)
 if (Helpers::hasOverride('t') === false) { // @codeCoverageIgnore
 	/**
 	 * Returns translate string for key from translation file
-	 *
-	 * @param string|array $key
-	 * @param string|null $fallback
-	 * @param string|null $locale
-	 * @return array|string|null
 	 */
-	function t($key, string $fallback = null, string $locale = null)
-	{
+	function t(
+		string|array $key,
+		string|null $fallback = null,
+		string|null $locale = null
+	): array|string|null {
```

Comment:
The type hints don't fully match `I18n::translate()`

Reply:
✅